extensions/wikia/Helios/HelperController.class.php

```diff
@@ -15,8 +15,10 @@ public function checkAntiSpoof()
 	{
 		$this->response->setFormat( 'json' );
 		$this->response->setCacheValidity( \WikiaResponse::CACHE_DISABLED );
+		$this->response->setVal( 'allow', false );
```

Comment:
@ArturKlajnerok — this one is not needed here as it is set in `return` statements.